File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.py
def solution(s):
    answer = True
    
    p_count = 0
    y_count = 0
    
    chars = list(s.lower())
    for char in chars:
        if char == 'p':
            p_count += 1
        elif char == 'y':
            y_count += 1
    
    if p_count != y_count:
        answer = False
    
    return answer

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.java
class Solution {
    boolean solution(String s) {
        boolean answer = true;
        int pCount = 0;
        int yCount = 0;
        s = s.toUpperCase();
        for (int i = 0; i < s.length(); i++) {
            if (s.charAt(i) == 'P') {
                pCount++;
            } else if (s.charAt(i) == 'Y') {
                yCount++;
            }
        } if (pCount != yCount) {
            answer = false;
        }
        return answer;
    }
}